### PR TITLE
load the certificate from the file system

### PR DESCRIPTION
**Pull request by pallenSL**

### CODE REVIEW

**Comment by jebbinSCL** [comment]
What do we need to do now to generate the certificate / secrets and run this locally?

**Comment by pallenSL** [comment]
You will need a pfx file, as it is no longer available to all via GIT and DockerHub.
secrets.config includes the name/path of the pfx file and it's password.

**Comment by jebbinSCL** [comment]
Could you also add a template for the secrets.config? I think right now people would need to read through the code to figure out what is required. 
Also, probably want to explicitly where the secrets.config and certificate should be placed.

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+---
+You can generate a self-signed certificate for use by Auth.
+
+On Windows Command Prompt use makecert:
+N.B.
+ "^" is the DOS line continuation character, just for clarity
+ CN= is "Common Name".  Just pick one.
+ -r is for self signed.
+
+> makecert.exe ^
```
Comment (jebbinSCL) [inline README.md]: Where is makecert.exe located?

Comment (pallenSL) [inline README.md]: It comes with the Windows SDKs.

**Comment by jebbinSCL** [comment]
Since the instructions are a separate thing I'll merge this for now. However, I think the instructions need expanding, since I haven't been able to run them locally yet.

**Pull request merged**